bin/index.js

```diff
@@ -8,8 +8,20 @@ const runReactAppBlueprint = require('../blueprints/react-app');
 
 const usage = chalk.cyan(`\nHi, I'm Gavin! 👋\nUsage: gavin [command] [options]`);
 
+// TODO: Would be nice to have a bundler handle syncing the volta version with this one
+const nodeMajor = 18;
+
+const checkNodeVersion = () => {
+    if (parseInt(process.version.split('.')?.[0]?.replace('v', '')) < nodeMajor) {
+        console.log(chalk.redBright('Command failed'))
+        console.log(chalk.redBright(`Gavin formally requests you upgrade to Node ${nodeMajor}+`))
+        process.exit(1)
+    }
+}
```

Comment:
lets move this out to `../utils/node.js` if we can. trying to keep the bin/index as lean as possible, as basically just the yargs entrypoint/defintion file.